core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.directive.html

```diff
@@ -51,14 +51,17 @@ <h3>Add Misconception</h3>
         <strong>Should a response to this misconception be enforced on every question linked to this skill?</strong>
         <input type="checkbox" ng-model="misconceptionMustBeAddressed">
       </div>
+      <div ng-if="isEditorContentLengthExceeded()" class="oppia-length-validation-error">
+        The content length is too long. Please use at most 1000 characters.
```

Comment:
Let's make this message clearer for the creator, who very likely doesn't know that we're setting a maximum on the number of chars in the HTML string.
```suggestion
        Please shorten the content (images, tabs, and other special items take up the most storage).
```

Reply:
Done